src/summary.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const chalk = require('chalk');
```

Comment:
Let's include a comment here that this is taken from eslint-summary, with a link to the github and a copy of its license.  Credit where it's due, and all.  :)